### ISSUE TITLE
:wheelchair: a11y(nav): add labels to the nav elements in the tabs

### ISSUE THREAD

**Issue by hirsch88** [baloise/design-system#1639]
### Description of this issue

there should be aria-labels on the nav tags to distinguish between "meta", "main" and "sub" navigations

### How to fix it

![Image](https://github.com/user-attachments/assets/4dd3183a-267f-44b1-a2e1-8fa89263def6)

1 Mainnavigation
2 Navigation
3 Subnavigation (falls benötigt in diesem Bereich
4 Inpage Navigation (nächster Screenshot)

### Code Reproduction URL

-

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by hirsch88** [comment]
waiting for translations
